### PR TITLE
Add Chinese doc

### PR DESCRIPTION
**Pull request by Zhanxueyou**
Signed-off-by: zhanxueyou <zhanxueyou@msn.com>

### CODE REVIEW

File: content/zh/docs/chart_template_guide/builtin_objects.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ weight: 3
   - `Release.Service`： 该service用来渲染当前模板。Helm里一般是`Helm`
 - `Values`： Values是从`values.yaml`文件和用户提供的文件传进模板的。`Values`默认为空
 - `Chart`： `Chart.yaml`文件内容。 `Chart.yaml`里的任意数据在这里都可以可访问的。比如 `{{ .Chart.Name }}-{{ .Chart.Version }}` 会打印出 `mychart-0.1.0`
-  - [Chart 指南](http://helm.sh/zh/docs/topics/charts.md#Chart-yaml-文件) 中列出了可用字段
-- `Files`： 在chart中提供访问所有的非特殊文件。当你不能使用它访问模板时，你可以访问其他文件。请查看这个部分 [文件访问](http://helm.sh/docs/chart_template_guide/accessing_files.md) 了解跟多信息
+  - [Chart 指南](http://docs.helm.sh/zh/docs/topics/charts.md#Chart-yaml-文件) 中列出了可用字段
```
Comment (bridgetkromhout) [inline content/zh/docs/chart_template_guide/builtin_objects.md:22]: Links don't need a `.md`

File: content/zh/docs/topics/chart_repository.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+该指南将介绍如何操作。
+
+## 先决条件
+
+* 先阅读[快速开始](https://docs.helm.sh/zh/docs/topics/quickstart/)
+* 阅读[Charts](https://docs.helm.sh/zh/docs/topics/charts.md/)文档
+
+## 创建一个chart仓库
+
+_chart仓库_ 是一个配置了`index.yaml`文件和一些已经打包chart的HTPP服务器。当你准备好分享chart时，最好的方法是将chart上传到chart仓库。
```
Comment (bridgetkromhout) [inline content/zh/docs/topics/chart_repository.md:19]: HTTP?

File: content/zh/docs/topics/library_charts.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+
+这些模板显示了如何从辅助模板中继承公共代码来将你的代码简化到资源的配置或自定义。
+
+为了能使用公共代码，我们需要添加一个`common`依赖。在`demo/Chart.yaml`文件最后啊添加以下内容： 
+
+```yaml
+dependencies:
+- name: common
+  version: "^0.0.5"
+  repository: "https://kubernetes-charts-incubator.storage.googleapis.com/"
```
Comment (bridgetkromhout) [inline content/zh/docs/topics/library_charts.md:323]: Please update the repository URLs, as seen in https://github.com/helm/helm-www/pull/897.

File: content/zh/docs/topics/registries.md [hunk 4]
```diff
@@ -217,6 +217,6 @@ $ cat ~/Library/Caches/helm/registry/cache/blobs/sha256/31fb454efb3c69fafe536725
 
 ## 从chart仓库迁移
 
-从经典 [chart 仓库](https://helm.sh/docs/topics/chart_repository.md)
+从经典 [chart 仓库](https://docs.helm.sh/zh/docs/topics/chart_repository.md)
```
Comment (bridgetkromhout) [inline content/zh/docs/topics/registries.md:220]: The correct URL will be https://helm.sh/zh/docs/topics/chart_repository/ although you can just say `/docs/topics/chart_repository/` too.

File: content/zh/docs/topics/version_skew.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+该文档描述了在Helm和Kubernetes之间支持的最大版本偏差。
+
+## 支持的版本
+
+Helm的版本用 `x.y.z` 描述，`x`是主版本，`y`是次版本，`z`是补丁版本，遵循
+[语义化版本](https://semver.org/spec/v2.0.0.html) 术语。
+
+Helm项目维护了一个针对最近次要版本的发布分支。适当的修复，包括安全修复、从发布分支中的cherry-pick，
+视严重程度而定。更多细节请查看[Helm版本策略](https://docs.helm.sh/docs/release_policy.md)。
```
Comment (bridgetkromhout) [inline content/zh/docs/topics/version_skew.md:14]: /docs/topics/release_policy/

File: content/zh/docs/topics/charts.md [hunk 6]
```diff
@@ -607,7 +607,7 @@ storage: "gcs"
 **注意：** 如果`helm install`或`helm upgrade`使用了`--set`参数，这些值在客户端会被简单地转换为YAML。
 
 **注意：** 如果values 文件存在任何必需的条目，它们会在chart模板中使用['required'
-函数](http://helm.sh/docs/howto/charts_tips_and_tricks.md) 声明为必需的。
+函数](http://docs.helm.sh/zh/docs/howto/charts_tips_and_tricks.md) 声明为必需的。
```
Comment (bridgetkromhout) [inline content/zh/docs/topics/charts.md:610]: Don't end with .md - this URL will be https://helm.sh/zh/docs/howto/charts_tips_and_tricks/ but /docs/howto/charts_tips_and_tricks/  is fine too.

Comment (Zhanxueyou) [reply content/zh/docs/topics/charts.md:610]: OK，I need to distinguish between translation，URL will be https://helm.sh/zh/docs/howto/charts_tips_and_tricks/

**Review by bridgetkromhout — APPROVED** [review]
Thank you, @Zhanxueyou!

**Pull request merged**